lib/u3d/cache.rb

```diff
@@ -31,11 +31,7 @@ module U3d
   class Cache
     using ::CoreExtensions::OperatingSystem
 
-    # Path to the directory containing the cache for the different OS
-    DEFAULT_LINUX_PATH = File.join(ENV['HOME'], '.u3d').freeze
```

Comment:
in theory this is breaking APIs :)

Reply:
Would it be better to have something like
```ruby
DEFAULT_LINUX_PATH = U3dCore::Helper.data_path
... (same for windows and mac)
```

Reply:
Let's say for this time we don't care.